LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Bernard
+Copyright (c) 2014-2016 Bernard
```

Comment:
Bernard Contributors, don't want to claim copyright over something that is build by the community.